packages/common/src/extensions/slickRowBasedEdit.ts

```diff
@@ -52,10 +59,8 @@ export class SlickRowBasedEdit {
   protected _editedRows: Map<string, EditedRowDetails> = new Map();
 
   private _existingEditCommandHandler: ((item: any, column: Column<any>, command: EditCommand) => void) | undefined;
-  private btnUpdateTitle: string = '';
-  private btnEditTitle: string = '';
-  private btnDeleteTitle: string = '';
-  private btnCancelTitle: string = '';
+  protected _currentLang = 'en';
```

Comment:
I meant this one.

Reply:
I know you meant that one but the reason to keep it under a variable is to avoid having to calling `getCurrentLanguage()` in the Formatter for every row of the dataset which was mainly the concern of this PR